contract-map.json

```diff
@@ -13,6 +13,22 @@
     "symbol": "HGT",
     "decimals": 8
   },
+
+  "0x8727c112C712c4a03371AC87a74dD6aB104Af768": {
+    "name": "Jetcoin Institute Token",
+    "logo": "JETCOIN28.png",
+    "erc20": true,
+    "symbol": "JET",
+    "decimals": 18
+  },
+  "0xf8e386EDa857484f5a12e4B5DAa9984E06E73705": {
+    "name": "Indorse Token",
+    "logo": "indorse2.png",
```

Comment:
Logos should be square, sorry this wasn't in the README. Please update the image to be used as a coin logo, ideally graphic only.

Reply:
@david-sg can you pass me a square logo?